File: VB/CustomHandlers/AppointmentDragInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraScheduler

Public Class AppointmentDragInfo
	Public Sub New()
	End Sub

	Public Property Appointment() As Appointment
	Public Property TimeAtCursor() As TimeSpan

	Public Property InitialInterval() As TimeInterval
End Class

File: CS/CustomHandlers/AppointmentDragInfo.cs
using System;
using DevExpress.XtraScheduler;

public class AppointmentDragInfo {
    public AppointmentDragInfo() { }

    public Appointment Appointment { get; set; }
    public TimeSpan TimeAtCursor { get; set; }

    public TimeInterval InitialInterval { get; set; }
}